contracts/Holdable.sol

```diff
@@ -327,10 +356,42 @@ contract Holdable is IHoldable, ERC20 {
         );
     }
 
-    function _decreaseHeldBalance(string memory operationId, uint256 value) internal {
+    function _decreaseHeldBalance(string memory operationId, uint256 value) private {
         Hold storage executableHold = holds[operationId.toHash()];
 
         heldBalance[executableHold.origin] = heldBalance[executableHold.origin].sub(value);
         _totalHeldBalance = _totalHeldBalance.sub(value);
     }
+
+    function _computeExpiration(uint256 _timeToExpiration) private view returns (uint256) {
+        uint256 expiration = 0;
+
+        if (_timeToExpiration != 0) {
+            /* solium-disable-next-line security/no-block-members */
+            expiration = now.add(_timeToExpiration);
+        }
+
+        return expiration;
+    }
+
+    function _checkHold(address to) private pure {
+        require(to != address(0), "Payee address must not be zero address");
+    }
+
+    function _checkHoldFrom(address to, address from) private view {
+        require(to != address(0), "Payee address must not be zero address");
+        require(from != address(0), "Payer address must not be zero address");
+        require(operators[from][msg.sender], "This operator is not authorized");
+    }
+
+    function _checkRenewableHold(string memory operationId) private view {
+        Hold storage renewableHold = holds[operationId.toHash()];
+
+        require(renewableHold.status == HoldStatusCode.Ordered, "A hold can only be renewed in status Ordered");
```

Comment:
We do not have any test about this check.

Reply:
Yes, it is tested:

[For renewHold](https://github.com/IoBuilders/holdable-token/blob/1d382398028b26b11124985e684137b32706c15c/test/Holdable.js#L1184)
[For renewHoldWithExpirationDate](https://github.com/IoBuilders/holdable-token/pull/4/files/1d382398028b26b11124985e684137b32706c15c#diff-e05a23bdf4e02b99f0bcb5e63cc07a49R1336)